packages/test/test-service-load/src/leaderElection.ts

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+import { IFluidDataStoreRuntime } from "@fluidframework/datastore-definitions";
+import { ISignalMessage } from "@fluidframework/protocol-definitions";
+
+export class LeaderElection {
+    private readonly dataStoreRuntime: IFluidDataStoreRuntime;
+    private readonly clientId: string | undefined;
+    private readonly logger: ITelemetryBaseLogger;
+    private leaderId: string | undefined;
+    private readonly beatInEveryNSecs: number = 1000; // 1 secs
+    private readonly leaderWait: number = 5000; // 5 secs
+    private lastPinged: Date | undefined;
+
+    constructor(dataStoreRuntime: IFluidDataStoreRuntime,
+        clientId: string | undefined,
```

Comment:
```suggestion
        private readonly clientId: string | undefined,
```
you can create members directly in the constructor, no need to specify and assign them separately

Reply:
looking at this further. i think the only thing this class needs is the IFluidDataStoreRuntime. it has the clientId and a logger. no need to take the other stuff seperately